CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## To Be Released
 
+## 5.0.0
+
```

Comment:
Could you specify in big what are the breaking changes? (Addition of Context and deleted methods?)